accounts-db/benches/utils.rs

```diff
@@ -0,0 +1,75 @@
+use {
+    rand::{
+        distributions::{Distribution, WeightedIndex},
+        Rng, SeedableRng,
+    },
+    rand_chacha::ChaChaRng,
+    solana_sdk::{
+        account::AccountSharedData, pubkey::Pubkey, rent::Rent,
+        rent_collector::RENT_EXEMPT_RENT_EPOCH,
+    },
+    std::iter,
+};
+
+/// Returns an iterator with storable accounts.
+pub fn accounts<'a>(
+    seed: u64,
+    data_sizes: &'a [usize],
+    weights: &'a [usize],
+) -> impl Iterator<Item = (Pubkey, AccountSharedData)> + 'a {
+    let distribution = WeightedIndex::new(weights).unwrap();
+    let mut rng = ChaChaRng::seed_from_u64(seed);
+    let rent = Rent::default();
+
+    iter::repeat_with(move || {
+        let index = distribution.sample(&mut rng);
+        let data_size = data_sizes[index];
+        let owner: [u8; 32] = rng.gen();
+        let owner = Pubkey::new_from_array(owner);
+        (
+            owner,
+            AccountSharedData::new_rent_epoch(
+                rent.minimum_balance(data_size),
+                data_size,
+                &owner,
+                RENT_EXEMPT_RENT_EPOCH,
+            ),
+        )
+    })
+}
+
+/// Returns an iterator over storable accounts such that the cumulative size of
+/// all accounts does not exceed the given `size_limit`.
+#[allow(dead_code)]
```

Comment:
With the current approach, it's necessary, because `benches/bench_accounts_file.rs` and `read_only_accounts_cache.rs` are including this file with `mod utils` separately. `bench_accounts_file` uses only the first function and that's enough to trigger the warning.

If we want to avoid that, an alternative would be to move this `utils` module to `src/bench_utils.rs` and introduce a feature flag.

Reply:
Alternatively, I can put that `#[allow(dead_code)]` at the top and add the comment explaining why it's needed.